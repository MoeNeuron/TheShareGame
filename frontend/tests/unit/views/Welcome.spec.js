import { mount } from '@vue/test-utils';
import Welcome from '@/views/Welcome.vue';

describe('Welcome.vue', () => {
  it('matches Snapshot', () => {
    expect(mount(Welcome)).toMatchSnapshot();
  });
});
