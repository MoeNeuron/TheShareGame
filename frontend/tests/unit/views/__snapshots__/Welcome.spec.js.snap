// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Welcome.vue matches Snapshot 1`] = `
<div>
  <header class="masthead text-white text-center">
    <div class="overlay"></div>
    <div class="container">
      <div class="row">
        <div class="col-xl-9 mx-auto">
          <h1 class="mb-5">Sign up now and join tons of other players</h1>
        </div>
        <div class="text-center">
          <div class="row col-lg-12"><a href="/register"><button type="button" class="btn btn-primary">Register Now!</button></a> <a href="/login"><button type="button" class="btn btn-primary">Log in</button></a></div>
          <div class="row col-lg-12" style="margin-top: 10px;"><a href=""><button type="button" class="btn btn-sm btn-primary">Forgot password?</button></a></div>
        </div>
      </div>
    </div>
  </header>
  <section class="features-icons bg-light text-center">
    <div class="container">
      <div class="row">
        <div class="col-lg-4">
          <div class="features-icons-icon d-flex"><i class="fa fa-4x fa-building m-auto text-primary"></i></div>
          <h3>Manage your Stock Company</h3>
          <p class="lead mb-0">
            From Marketing, Finance to Investing. You have to do it all
          </p>
        </div>
        <div class="col-lg-4">
          <div class="features-icons-icon d-flex"><i class="fa fa-4x fa-handshake m-auto text-primary"></i></div>
          <h3>No Pay-To-Win</h3>
          <p class="lead mb-0">
            Money won't help you. Only the smartest will win
          </p>
        </div>
        <div class="col-lg-4">
          <div class="features-icons-icon d-flex"><i class="fa fa-4x fa-chart-line m-auto text-primary"></i></div>
          <h3>Group up with Players</h3>
          <p class="lead mb-0">
            Chat with others, write newspaper articles or join a fond
          </p>
        </div>
      </div>
    </div>
  </section>
  <section class="showcase">
    <div class="container-fluid p-0">
      <div class="row no-gutters">
        <div class="col-lg-6 col-sm-12"><img src="" style="width: 100%;"></div>
        <div class="col-lg-6 order-lg-1 my-auto showcase-text">
          <h2>Analyze Shares</h2>
          <p class="lead mb-0">
            Analyze the key figures of other Shares and evaluate whether they are overpriced or
            not
          </p>
        </div>
      </div>
      <div class="row no-gutters">
        <div class="col-lg-6 my-auto showcase-text">
          <h2>Orders</h2>
          <p class="lead mb-0">

            Jump in the live order section to check out the all orders.
            Create normal, dynamic and iceberg orders

          </p>
        </div>
        <div class="col-lg-6 text-white showcase-img"></div>
      </div>
      <div class="row no-gutters">
        <div class="col-lg-6 col-sm-12"><img src="" style="width: 100%;"></div>
        <div class="col-lg-6 order-lg-1 my-auto showcase-text">
          <h2>Earn Achievements</h2>
          <p class="lead mb-0">
            While playing you can earn several achievements with different degrees of difficulty.
            Will you be able to collect'em all?
          </p>
        </div>
      </div>
    </div>
  </section>
</div>
`;
